src/pages/RoomMembersPage.tsx

```diff
@@ -182,12 +184,14 @@ function RoomMembersPage({report, session, policies}: RoomMembersPageProps) {
                 return;
             }
             const pendingChatMember = report?.pendingChatMembers?.findLast((member) => member.accountID === accountID.toString());
+            const isAdmin = !!(policy && policy.employeeList && details.login && policy.employeeList[details.login]?.role === CONST.POLICY.ROLE.ADMIN);
+            const isDisabled = (isPolicyExpenseChat && isAdmin) || accountID === session?.accountID || pendingChatMember?.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE;
```

Comment:
Regarding this issue https://github.com/Expensify/App/issues/42928, when determining if the option should be disabled, we should have also checked the workspace chat's ownerAccountID and disable it when account id matches the ownerAccountID. 

We [fixed](https://github.com/Expensify/App/pull/44595) this by adding a check `details.accountID === report.ownerAccountID;`